metadata.rb

```diff
@@ -3,7 +3,7 @@
 maintainer_email 'foss@gilles.cornu.name'
 license 'Apache 2.0'
 description 'Installs Google Android SDK'
-version '0.2.0'
+version '0.2.1'
```

Comment:
Oops my bad 😬 sorry! Better that I wait for this afternoon, when I'll really have time to focus on review/test work ;-)
See you later then... Thanks again!